apis/python/src/tiledbsoma/_dataframe.py

```diff
@@ -376,6 +376,31 @@ def index_column_names(self) -> Tuple[str, ...]:
         """
         return self._tiledb_dim_names()
 
+    def get_enumeration_values(
+        self: DataFrame, column_names: Sequence[str]
+    ) -> dict[str, pa.Array]:
+        """Returns an Arrow array of the specified columns'
+        enumeration/dictionary/categorical values. Raises ``ValueError`` if any
+        of the the specified column names is not in the schema, or if any is not
+        of Arrow dictionary type."""
+        self._check_open_read()
```

Comment:
why is this only available for `read`?  it is useful in both modes, the the `tiledb-py` API allows it to be fetched in both cases.

It is essentially schema information, and should be available in all the same modes that schema is available (i.e., read and write)